pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -64,7 +64,7 @@ const (
 	tprName           = "mon.rook.io"
 	fsidSecretName    = "fsid"
 	monSecretName     = "mon-secret"
-	adminSecretName   = "admin-secret"
+	adminSecretName   = "admin"
```

Comment:
The integration tests are failing because the toolbox pod can't load the admin secret that was renamed here. Remind me why it needs to be renamed? If it needs to be renamed, I'd suggest copying it here and having it in both places for backwards compatibility.

Reply:
ceph-csi looks for a secret k/v where key is the ceph id and value is the keyring.